lib/rt-thread/tusb_config.h

```diff
@@ -113,6 +113,7 @@ extern "C" {
 //--------------------------------------------------------------------
 // DEVICE CONFIGURATION
 //--------------------------------------------------------------------
+#ifdef PKG_TINYUSB_DEVICE_ENABLE
 #if defined(PKG_TINYUSB_DEVICE_ENABLE)
```

Comment:
You can remove this `#if defined(PKG_TINYUSB_DEVICE_ENABLE)`  as `PKG_TINYUSB_DEVICE_ENABLE` already tested.

Reply:
already edited